CHANGELOG.md

```diff
@@ -14,12 +14,17 @@ Our versioning strategy is as follows:
 
 ### 🎉 New Features & Improvements
 
-* `[create-sitecore-jss]`: Refactoring/Cleanup for scss files in SXA components ([#119](https://github.com/Sitecore/content-sdk/pull/119))([#122](https://github.com/Sitecore/content-sdk/pull/122))
+* `[create-app]`: Refactoring/Cleanup for scss files in SXA components ([#119](https://github.com/Sitecore/content-sdk/pull/119))([#122](https://github.com/Sitecore/content-sdk/pull/122))
 * `[core]` `[nextjs]` [DesignLibrary] Include metadata in the Design Library rendering mechanism ([#118](https://github.com/Sitecore/content-sdk/pull/118))
 * `[core]` `[nextjs]` `[cli]` Add automatic component map generation ([#124](https://github.com/Sitecore/content-sdk/pull/124) [#128](https://github.com/Sitecore/content-sdk/pull/128) [#130](https://github.com/Sitecore/content-sdk/pull/130))
 
 ### 🛠 Breaking Changes
 
+* `[all]` Renamed all JSS references to Content SDK across the codebase: ([#131](https://github.com/Sitecore/content-sdk/pull/131))
+  - The create-sitecore-jss package has been renamed to create-app
+  - Component types and props renamed (e.g. `JssComponent` to `ContentSdkComponent`)
+  - Updated placeholder names in test data from `jss-test` to `content-sdk-test`
+  - Documentation references have also been updated to reflect the new content-sdk
```

Comment:
there's no need to list changes to unit test and monorepo doc files in changelog. Let's only list the changes that affect the end app. I.e. `imports JSS1, JSS2, JSS3 from package xyz have been renamed to ContentSDK1, 2, 3`.
Please also provide an example for new `create-app` command.